src/VecSim/vec_sim_common.h

```diff
@@ -124,12 +127,13 @@ typedef struct {
 typedef struct {
     union {
         struct {
-            size_t indexSize;        // Current count of vectors.
-            size_t blockSize;        // Sets the amount to grow when resizing
-            size_t M;                // Number of allowed edges per node in graph.
-            size_t efConstruction;   // EF parameter for HNSW graph accuracy/latency for indexing.
-            size_t efRuntime;        // EF parameter for HNSW graph accuracy/latency for search.
-            size_t max_level;        // Number of graph levels.
+            size_t indexSize;      // Current count of vectors.
+            size_t blockSize;      // Sets the amount to grow when resizing
+            size_t M;              // Number of allowed edges per node in graph.
+            size_t efConstruction; // EF parameter for HNSW graph accuracy/latency for indexing.
+            size_t efRuntime;      // EF parameter for HNSW graph accuracy/latency for search.
```

Comment:
Yes I understand that. I believe that this is the right way to do it, no? Also, Since range query is planned for RediSearch 2.6 where we are adding our optional hybrid queries parameters, I guess we can also add the `EPSILON` optional parameter as well.